lib/show.rb

```diff
@@ -0,0 +1,14 @@
+class Show
+  attr_reader :name, :creator, :characters
+  
+  def initialize(name, creator,characters)
+    @name = name
+    @creator = creator
+    @characters = characters
+  end
+
+  def total_salary
+    @characters[0].salary + @characters[1].salary
+  end
```

Comment:
I guess this would work fine if we know we'll only ever have two characters on a TV show ... how would you do this if we had only 1 character, or 50?